brainkeeper/src/app/app.component.html

```diff
@@ -1 +1,8 @@
 <router-outlet></router-outlet>
+
+
```

Comment:
These lines were removed in a previous commit, please remove them again

Reply:
changed with commit: 3c092315bd02c68198ed286b95baee75aafa7339